src/main/resources/templates/modulbeauftragter.html

```diff
@@ -53,8 +56,126 @@
+                        </div>
+
+                        <div class="mt-3 mb-3">
+                            <label for="title">Titel (Englisch)</label>
+                            <input type="text"
+                                   style="float: right"
+                                   name="titleEnglish"
+                                   class="form-control col-md-8 col-sm-8"
+                                   id="titleEnglish"
+                                   placeholder="Englischer Titel der Veranstaltung" required>
+                        </div>
+
+                        <div class="mt-3 mb-3">
+                            <label for="organizer">Modulbeauftragter</label>
+                            <input type="text"
+                                   style="float: right"
+                                   name="organizer"
+                                   class="form-control col-md-8 col-sm-8"
+                                   id="organizer"
+                                   placeholder="Modulbeauftragter der Veranstaltung" required>
+                        </div>
+
+                        <div class="mt-3 mb-3">
+                            <label for="cp">Creditpoints</label>
+                            <input type="number"
+                                   style="float: right"
+                                   name="cp"
+                                   class="form-control col-md-8 col-sm-8"
+                                   id="cp"
+                                   placeholder="CP der Veranstaltung" required>
+                        </div>
+
+                        <div class="mt-3 mb-3">
+                            <label for="modulcategory">Modulkategorie</label>
+                            <select name="modulcategory"
+                                    style="float: right"
+                                    class="form-control col-md-8 col-sm-8"
+                                    id="modulcategory" required>
+                                <option selected hidden value="">Modulkategorie Wählen</option>
+                                <option value="PFLICHT_INFO">Pflichtmodul der Informatik</option>
+                                <option value="PFLICHT_MATHE">Pflichtmodul der Mathematik</option>
+                                <option value="NEBENFACH">Nebenfachmodul</option>
+                                <option value="PRAXIS">Praxis- und Berufsorientierung</option>
+                                <option value="WAHLPFLICHT_BA">Wahlpflichtmodul (B.Sc.)</option>
+                                <option value="WAHLPFLICHT_MA">Wahlpflichtmodul (M.Sc.)</option>
+                                <option value="BACHELORARBEIT">Bachelorarbeit</option>
+                                <option value="PROJEKTARBEIT">Projektarbeit</option>
+                                <option value="MASTERARBEIT">Masterarbeit</option>
```

Comment:
SONSTIGE fehlt als Kategorie

Reply:
Hab ich jetzt hinzugefügt; wird in der nächsten Version vorhanden sein.